app/src/main/res/layout/fragment_video_preview.xml

```diff
@@ -25,18 +25,63 @@
                 android:layout_width="wrap_content"
                 android:layout_height="wrap_content"
                 android:layout_alignParentStart="true"
-                android:layout_alignParentLeft="true"
                 android:layout_alignParentTop="true"
                 android:textColor="@color/text_main"
                 android:textSize="24sp" />
 
+            <TextView
+                android:id="@+id/textSubtitles"
+                android:layout_width="match_parent"
+                android:layout_height="wrap_content"
+                android:layout_below="@+id/textTitle"
+                android:ellipsize="marquee"
+                android:singleLine="true"
+                android:textColor="@color/text_second"
+                android:textSize="14sp"
+                android:visibility="gone" />
+
+            <TextView
+                android:id="@+id/headerTextPreviewDescription"
+                android:layout_width="fill_parent"
+                android:layout_height="48dp"
+                android:layout_below="@+id/textSubtitles"
+                android:layout_alignParentStart="true"
+                android:background="@color/transparent"
+                android:fontFamily="sans-serif-medium"
+                android:gravity="center_vertical"
+                android:text="@string/title_section_description"
+                android:textColor="@color/apptheme_secondary"
+                android:textSize="14sp" />
+
+            <TextView
+                android:id="@+id/textPreviewDescription"
+                android:layout_width="wrap_content"
+                android:layout_height="wrap_content"
+                android:layout_below="@+id/headerTextPreviewDescription"
+                android:layout_alignParentStart="true"
+                android:ellipsize="marquee"
+                android:foreground="@drawable/gradient_light_to_transparent_from_bottom"
```

Comment:
What should I do about this? This means that everybody with an API level below 23 will not see a fade in the description, if it's collapsed. Is that that bad? I dont think so.

Reply:
Fine for me, Android 5 is from 2014.

Reply:
Yep, that's because pre-23 `android:foregound` could only be applied to `FrameLayout`. If you want to fix this, you could wrap that code inside a FrameLayout or add a simple `View` with the fade as its background atop the TextView.

Reply:
I will implement the FrameLayout solution 😃 it will also get rid of the Feature.FOREGROUND checks